packages/flutter/lib/src/rendering/sliver_group.dart

```diff
@@ -168,3 +168,147 @@ bool _assertOutOfExtent(double extent) {
   }
   return true;
 }
+
+/// A sliver that places multiple sliver children in a linear array along the
+/// main axis.
+///
+/// Typically, the slivers will be laid out one at a time. Slivers that have been
+/// scrolled past partially or entirely will be provided a nonzero scrollOffset that
+/// informs them at which point along the main axis should the sliver children
+/// start painting themselves.
+///
+/// For other slivers, we lay them out with a [SliverConstraints.remainingPaintExtent]
+/// that represents how much space in the viewport the sliver has.
+///
+/// Finally, this sliver will also ensure that all slivers are painted within
+/// the total layout extent of the sliver by painting slivers that are "out of
+/// bounds" with a negative [SliverPhysicalParentData.paintOffset].
```

Comment:
I think I see now. It is  bit difficult to discern which sliver you are referring to, the group as a whole or its children, can you clarify?

Reply:
I clarified this point.